packages/event-status-service/src/controllers/automation/get.ts

```diff
@@ -0,0 +1,44 @@
+import { Response } from 'express'
+import { logger } from '../../core/logger'
+import { AppRequest, stringifyError } from '@reapit/node-utils'
+import { db } from '../../core/db'
+import { generateAutomationItem } from '../../schemas/automation.schema'
+
+export default async (req: AppRequest, res: Response) => {
+  const id = req.params.id as string | undefined
+  const { traceId } = req
+
+  try {
+    logger.info('Getting automation by id...', { traceId, id })
+
+    const itemToGet = generateAutomationItem({ id })
+    const result = await db.get(itemToGet)
+
+    if (result.clientCode !== (req as any).user.clientCode) {
```

Comment:
Seeing this again, is it possible to extend AppRequest a generic eg AppRequest<{user: Foo>> so you don't have to cast it as any?